questions/Changing HTML Attribute dynamically

```diff
@@ -0,0 +1,20 @@
+### Can we modify an attribute's value dynamically for an HTML tag? 
+
+#### Answer
+
+Yes. It is possible by using javascript using 'getElementById' method.
+'<input type=“text” id=“inputField”>
+document.getElementById(“inputField”).attr(“type”, “password”);'
+Here we are changing type from 'text' to 'password'
+
+#### Good to hear
+
+* We can also use jQuery for this purpose using setAttribute() method.
```

Comment:
DO NOT mention jQuery. Seriously. A lot of companies will not hire you if you even use it or mention it as a possibility. While it's a powerful tool, it is also a technology the web wants to slowly fade out as it's different versions are broken in today's and the future's environments.

Reply:
While it is still powerful and useful, unless you need to use it in an interview... don't.